### ISSUE TITLE
schemix generate absolute path for names in Windows 10

### ISSUE THREAD

**Issue by tsanyqudsi** [ridafkih/schemix#46]
How to reproduce : 
- use Windows 10
- clone the `standard-usage` example from the repo
- install latest schemix
- change import from `../../../dist` to `schemix`
- npm start 

result : 
```
enum D:\Absolute\Path\on\Windows\Status {
  PENDING
  LIVE
  DELETED
  REMOVED
}
```

expected result : 
```
enum Status {
  PENDING
  LIVE
  DELETED
  REMOVED
}
```

**Comment by tsanyqudsi** [comment]
workaround : 
instead of using 
```
export default createModel((Model) => {
	Model.string('name')
		.string('descriptions')
		.mixin(Mixin);
});
```
use
```
export default createModel('modelName', (Model) => {
	Model.string('name')
		.string('descriptions')
		.mixin(Mixin);
});
```

to override the value.

**Comment by ridafkih** [comment]
@tsanyqudsi Definitely a bug, and a high priority one at that. Went out and got a windows system to use temporarily in order to help me debug and resolve this issue. Thanks so much.

**Comment by ridafkih** [comment]
@tsanyqudsi I'm travelling at the moment, would it be possible to get you to test #47 for me? Thanks in advance! If not, I'll be arriving back home in about a week and will be able to test then. :)

**Comment by ridafkih** [comment]
Fix released in v1.8.1! Thanks for the report and sorry for the wait!
